### PR TITLE
Added AutoMapper

### PR DESCRIPTION
**Pull request by MirazamTUT**

### CODE REVIEW

**Review by qAziz060708 — DISMISSED** [review]
Good Job

File: OnlinePharmacyShopping/PharmacyShopping.DataAccess/Models/ReportMedicine.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    {
+        public int ReportMedicineId { get; set; }
+
+        public int ReportId { get; set; }
+
+        public string MedicineId { get; set; }
+
+
+        public Medicine Medicine { get; set; }
+        public Report Report { get; set; }
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.DataAccess/Models/ReportMedicine.cs:14]: Bitta qator tashash kk edi

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

**Comment by isfandiyoro5** [comment]
> 



> Good Job

Qanaqa Good job

File: OnlinePharmacyShopping/PharmacyShopping.DataAccess/Repository/Repositories/ReportRepository.cs [hunk 2]
```diff
@@ -29,13 +29,13 @@ public async Task<int> DeleteReportAsync(Report report)
...
-            .Include(u => u.Medicines)
+            .Include(u => u.Customer)
+            .Include(u => u.ReportMedicines)
             .ToListAsync();
 
         public async Task<Report> GetReportByIdAsync(int id) => await _context.Reports
-            .Include(u => u.Customers)
-            .Include (u => u.Medicines)
+            .Include(u => u.Customer)
+            .Include (u => u.ReportMedicines)
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.DataAccess/Repository/Repositories/ReportRepository.cs]: Shu joyda 
.Include (u => u.ReportMedicines)
shuni shunga ozgartrish kk
.Include(u => u.ReportMedicines)

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

**Review by isfandiyoro5 — APPROVED** [review]

**Pull request merged**